server.go

```diff
@@ -2187,7 +2218,24 @@ func (s *server) inboundPeerConnected(conn net.Conn) {
 // manager of the attempt.
 func (s *server) outboundPeerConnected(c *connmgr.ConnReq, conn net.Conn) {
 	sp := newServerPeer(s, c.Permanent)
-	p, err := peer.NewOutboundPeer(newPeerConfig(sp), c.Addr.String())
+
+	// Just an alias.
+	peerAddr := c.Addr.String()
```

Comment:
Perhaps we should normalize a bit here and _only_ track the IP? IIUC the port would be included here.